website/src/components/DashboardHeader/DashboardHeader.jsx

```diff
@@ -1,40 +1,72 @@
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
 import {
   Layout,
 } from 'antd';
 import PropTypes from 'prop-types';
-import firebase from 'firebase/app';
-import 'firebase/auth';
 import { ProfileInfo, HeaderIcons } from '..';
 import './DisplayHeader.css';
 
 const { Header } = Layout;
 
-class DashboardHeader extends Component {
+class DashboardHeader extends PureComponent {
   static propTypes = {
     showDrawer: PropTypes.func,
+    firebase: PropTypes.shape({}),
```

Comment:
Tests don't work. Firebase needs to be initialized in order to work.  Since we are running unit tests each component gets tested separately. If a component relies on firebase to function the tests wont work because we didn't initialize firebase on a case by case basis. The Firebase tests only work because we give it an alternative for the firebase functions that cant be initialized. 
![image](https://user-images.githubusercontent.com/4605522/48727622-c7244c00-ec00-11e8-98c8-b5648a3629c5.png)